### DISCUSSION TITLE
Is there a way to bind button to switching playlist item in syncplay?

### DISCUSSION THREAD

**Discussion by soredake** [Syncplay/syncplay#680] [Q&A] [answered]
Due to the bug https://github.com/iamkroot/trakt-scrobbler/issues/202 I need to manually switch playlist item in syncplay, which is tiring.

Is there a way to, like, bind playlist switching like this "shift+j       script-binding syncplay/playlist-item-next"?

**Comment by Et0h** [comment, marked as answer]
> Due to the bug https://github.com/iamkroot/trakt-scrobbler/issues/202 I need to manually switch playlist item in syncplay, which is tiring.
> 
> Is there a way to, like, bind playlist switching like this "shift+j       script-binding syncplay/playlist-item-next"?

If you are using mpv then you can use the mpv bind system to bind a key to something like:
```mp.command('print-text "<chat>/qn</chat>"')```

Or just press enter, then write /qn, then press enter again.

See "Commands accessible via the chat facility" section of https://syncplay.pl/guide/client/ for the full list of commands, or just put /? In the chat system.

**Reply by soredake** [reply]
Thank you!
